tests/FSharp.Compiler.Service.Tests/ProjectAnalysisTests.fs

```diff
@@ -126,6 +126,7 @@ let ``Test project1 and make sure TcImports gets cleaned up`` () =
     let weakTcImports = test ()
     checker.InvalidateConfiguration Project1.options
     checker.ClearLanguageServiceRootCachesAndCollectAndFinalizeAllTransients()
+    GC.Collect()
```

Comment:
?

Reply:
Ah, this sneaked in. Unrelated to this PR, but this test was still flaky for me. Although `ClearLanguageServiceRootCachesAndCollectAndFinalizeAllTransients` also does call `GC.Collect()` it seems it is not enough.